worker/operations.py

```diff
@@ -550,7 +550,7 @@ def data_card(*args, **kwargs):
     }
 
     url = f"{MIT_API}/cards/get_data_card"
-    logger.info(f"Sending dataset {dataset_id} to MIT service at {url}")
+    logger.info(f"Sending dataset {dataset_id} and document {document_id} to MIT service at {url}")
```

Comment:
One thing to note is that this `{document_id}` will just be `None`. It won't break things though